### PR TITLE
Update apache commons-lang (2) to current commons-lang3

### PR DESCRIPTION
**Pull request by ebariaux**
Updated from commons-lang (2) to commons-lang3.
This is required to fix an issue with the Artifactory Jenkins plugin, see https://github.com/jfrog/jenkins-artifactory-plugin/pull/567

### CODE REVIEW

File: docker-java/template.mf [hunk 1]
```diff
@@ -13,7 +13,7 @@ Import-Template:
  javax.ws.rs.*;version="[2.0.0, 2.1.0)",
  org.apache.commons.compress.*;version="${commons-compress.version:short}",
  org.apache.commons.io.*;version="${commons-io.version:short}",
- org.apache.commons.lang.*;version="${commons-lang.version:short}",
+ org.apache.commons.lang33.*;version="${commons-lang3.version:short}",
```
Comment (bsideup) [inline docker-java/template.mf:16]: Shouldn't this be?
```suggestion
 org.apache.commons.lang3.*;version="${commons-lang3.version:short}",
```

Comment (ebariaux) [reply docker-java/template.mf]: Yes, of course, unfortunate search/replace

**Comment by bsideup** [comment]
@ebariaux merged, thanks! 💯

**Comment by Or-Geva** [comment]
@bsideup, is there any ETA when this commit will be released?

**Comment by ebariaux** [comment]
If you don't mind waiting for a few hours, I should have another PR

**Comment by ebariaux** [comment]
Done, see https://github.com/docker-java/docker-java/pull/1759

**Comment by yahavi** [comment]
Hi @ebariaux,
Currently `commons-lang2` causes many issues for us, for example: https://github.com/jfrog/jenkins-artifactory-plugin/issues/597
We'd like to get the next release of this library, to include this change.
Any ETA for the next release?
Thanks!

**Comment by ebariaux** [comment]
Hello @yahavi,
I'm not the maintainer of this project and don't know when a release is planned.
I just did create the PR.
I was facing the same issues with the Artifactory Plugin, I did updates in all the dependencies I could identify and created PRs for those changes.
By using all the latest versions (same code as those PRs), I was able to build an version of the Artifactory Plugin that solved all ours issues.

**Comment by yahavi** [comment]
@ebariaux sorry.
The question was meant to @bsideup.

**Comment by staticdev** [comment]
Same problem, need a release... @bsideup?

**Comment by glevy** [comment]
Hello @bsideup, 
Any ETA for the next release?

**Pull request merged**